mscore/plugin/pluginCreator.cpp

```diff
@@ -578,17 +565,7 @@ void PluginCreator::msg(const QString& s)
 
 void PluginCreator::showManual()
       {
-      if (helpBrowser == 0) {
-            helpBrowser = new HelpBrowser;
-            manualDock = new QDockWidget(tr("Manual"), 0);
-            manualDock->setObjectName("Manual");
-
-            manualDock->setWidget(helpBrowser);
-            Qt::DockWidgetArea area = Qt::RightDockWidgetArea;
-            addDockWidget(area, manualDock);
-            helpBrowser->setContent(manualPath());
-            }
-      manualDock->setVisible(!manualDock->isVisible());
+      QDesktopServices::openUrl(QUrl("https://dmitrio95.github.io/plugins/html/index.html"));
```

Comment:
Could you please replace the URL with this one?
https://musescore.github.io/MuseScore_PluginAPI_Docs/plugins/html/index.html
It looks like MuseScore-related domain is a better choice for permanent links like this :slightly_smiling_face:

Reply:
Final cleanup before you leave the org I guess. Yes, should be done this way, in any case

Reply:
done